lib/tarantool_server.py

```diff
@@ -1109,12 +1138,9 @@ def wait_until_started(self, wait_load=True):
         color_log('DEBUG: [Instance {}] Waiting until started '
                   '(wait_load={})\n'.format(self.name, str(wait_load)),
                   schema='info')
-
         if wait_load:
-            msg = 'entering the event loop|will retry binding|hot standby mode'
-            p = self.process if not self.gdb and not self.lldb else None
-            self.logfile_pos.seek_wait(msg, p, self.name)
-        while True:
+            self.wait_load(deadline)
+        while time.time() < deadline:
```

Comment:
I've tried your patches and once got the following:
```
[036] replication-py/init_storage.test.py                             
[036] Test.run() received the following error:
[036] Traceback (most recent call last):
[036]   File "/home/ubuntu/tarantool/test-run/lib/test.py", line 190, in run
[036]     self.execute(server)
[036]   File "/home/ubuntu/tarantool/test-run/lib/tarantool_server.py", line 412, in execute
[036]     exec(code, new_globals)
[036]   File "replication-py/init_storage.test.py", line 76, in <module>
[036]     replica.wait_until_started()
[036]   File "/home/ubuntu/tarantool/test-run/lib/tarantool_server.py", line 1137, in wait_until_started
[036]     while time.time() < deadline:
[036] TypeError: '<' not supported between instances of 'float' and 'NoneType'
[036] 
[036] [ fail ]
[036] Test failed! Output from reject file var/rejects/replication-py/init_storage.reject:
[036] box.schema.user.grant('guest', 'replication')
```

Reply:
Fixed.